src/app/base/components/dropdown/dropdown.component.scss

```diff
@@ -6,6 +6,10 @@
     width: 384px !important;
 }
 
+.tooltip_max_width_150::before {
```

Comment:
why is tooltip here in the name? Could this class not be used for every element and not only tooltips?

Reply:
This was a little bit tricky to add a max width to the tooltip, so we decided to go with class names that have different max width and those field is also part of the dropdown options. The name of the class is passed as string and that variable is part of the style of the element.

Reply:
Ah okay that makes sense. This can of course stay as it is but still I am confused why it says ".tooltip_max_width_150::before"  and not ".max_width_150::before", so it could be reused for other components. Or is it that the name itself gets compared in some way?

Reply:
These are hard coded values that we use only for the tooltips (the before selector is the tooltip that pops up). We could of course add them in the global style and 'reuse' the class on other places, that wouldn't be wrong, but there is no need for that.
If it was a class 'max_width_150' and we have the max width: 150px on multiple places it would make sense to have it as part of the global styling (this could be solved by tailwind classes or in-line style -> we use combination of these two). Adding a class would make sense if we have multiple properties that we want to add to a specific element (also can be solved by multiple classes in Tailwind -> how we do it). But I wouldn't add a class with ::before in the global stying unless it is used everywhere. 
Also this approach with having the size (150) in the name of the class is only used here for the tooltips on the dropdown, so we can differentiate the classes. In other cases I would go with more descriptive class names.

Reply:
Alright, thanks for the explanation, I think I got it :)